english/eating-walls/light-fire/fire.md

```diff
@@ -0,0 +1,4 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
@williamfbaur if you would like to only keep `make-some-smores.md`, you will need to update the pull request to remove this file from it. As is, if I merge this pull request then both files would be merged. To do this, you can remove the file from your local repository using the `git rm` command, then push the changes to your repository on GitHub to update the pull request.